app/src/App.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
+import Home from "./pages/home/home";
```

Comment:
Can you please separate imports (libraries - one group, internal code - the 2d one)

```
import React from "react";
import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";

import NotesContainer from "./pages/myNotes/NotesContainer";
import styles from "./styles.module.css";
```